### PR TITLE
feat(frontend): redirect to advanced ws url

### PR DESCRIPTION
**Pull request by MariePrandstetter**
## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [ ] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
  - I will create an issue for the failing functional tests 
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [ ] Manual, quality tests have been done

Ticket: #5093

### CODE REVIEW

File: frontend/src/container/NotificationWall.jsx [hunk 1]
```diff
@@ -95,7 +95,7 @@ export class NotificationWall extends React.Component {
   }
 
   getNotificationDetails = notification => {
-    const { props } = this
+    const { props, state } = this
```
Comment (Haskaris) [inline frontend/src/container/NotificationWall.jsx:98]: Just to understand, did you added `state` for consistency?
Because you don't seem to use it

Comment (MariePrandstetter) [reply frontend/src/container/NotificationWall.jsx:98]: If I'm not using it I probably just forgot to remove it, thanks!

**Review by Haskaris — APPROVED** [review]

File: frontend/src/component/Workspace/OpenWorkspaceAdvanced.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        },
+        props.user,
+        findUserRoleIdInWorkspace(props.user.userId, props.currentWorkspace.memberList, ROLE_LIST),
+        { ...props.currentWorkspace, workspace_id: props.currentWorkspace.id }
+      )
+    }
+  }
+
+  componentDidMount () {
+    console.log('%c<OpenContentApp> did Mount', 'color: #dcae84', this.props)
```
Comment (gbonaspetti) [inline frontend/src/component/Workspace/OpenWorkspaceAdvanced.jsx:34]:
```suggestion
    console.log('%c<OpenWorkspaceAdvanced> did Mount', 'color: #dcae84', this.props)
```

File: frontend/src/component/Workspace/OpenWorkspaceAdvanced.jsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    this.openWorkspaceAdvanced()
+  }
+
+  componentDidUpdate (prevProps) {
+    console.log('%c<OpenContentApp> did Update', 'color: #dcae84', this.props)
```
Comment (gbonaspetti) [inline frontend/src/component/Workspace/OpenWorkspaceAdvanced.jsx:39]:
```suggestion
    console.log('%c<OpenWorkspaceAdvanced> did Update', 'color: #dcae84', this.props)
```

**Review by gbonaspetti — CHANGES_REQUESTED** [review]

**Comment by gbonaspetti** [comment]
Bug during tests: 
- Click on Space Settings
- Click on another space in the sidebar
- The url changes but the app doesn't close
Same happens for a notification
- Click on a "request access" notification, it redirects to space settings
- Click on another space notification that should redirects to the dashboard
- The url changes but the app doesn't close

I'm gonna put the issue in "In progress".

**Comment by MariePrandstetter** [comment]
:warning: This PR was wrongly merged into maintenance/4.0
a revert commit will be pushed on maintenance/4.0

**Pull request merged**